bin/functions.pl

```diff
@@ -214,7 +216,7 @@ END
 	$ENV{'PATH'}      = File::Spec->catfile($JAVA_HOME, 'bin') . $PATHSEP . $ENV{'PATH'};
 
         my ($shortversion) = get_version_from_java(File::Spec->catfile($JAVA_HOME, 'bin', 'java'));
-        if ($shortversion >= 9) {
+        if ($shortversion >= 9 && $shortversion < 11) {
```

Comment:
Doesn't this basically only set the variable if JDK is 9 or 10?  What's the reason for keeping this if minversion=11 anyway?

Reply:
Good point. Removed.